### PR TITLE
Add Navigation Back Implementation

### PR DESCRIPTION
**Pull request by oandrz**
Hey man, hope you are doing well

Thank you for the template, it's very helpful for me personally to start a new project so I don't need to build everything from the scratch.

Anyway just want to propose some idea, maybe we could add implementation for back, here's what I'm thinking, feel free tell me what you think about this

Thanks

### CODE REVIEW

File: core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt [hunk 1]
```diff
@@ -20,4 +20,14 @@ class NavigationManager @Inject constructor(
             navigationCommandChannel.send(command)
         }
     }
+
+    fun back() {
+        externalMainImmediateScope.launch {
+            externalMainImmediateScope.launch {
```
Comment (krzdabrowski) [inline core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt:26]: I think these scopes and coroutines are duplicated 🙂

File: core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt [hunk 2]
```diff
@@ -20,4 +20,14 @@ class NavigationManager @Inject constructor(
             navigationCommandChannel.send(command)
         }
     }
+
+    fun back() {
```
Comment (krzdabrowski) [inline core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt:24]: I'd suggest using a verb for methods' names so something like `goBack` would be better 🙂

**Review by krzdabrowski — COMMENTED** [review]
Thanks for your contribution, I've left two comments 🙂

File: core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt (hunk 1 above)
Comment (oandrz) [reply core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt:26]: hey you're right, can't believe I miss this, thanks for spotting this 😃

File: core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt (hunk 2 above)
Comment (oandrz) [reply core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt:24]: hmm I think that's a great idea, let me add the fix

**Comment by oandrz** [comment]
Hey @krzdabrowski thanks for the feedback I just pushed a changes based on your feedback please review again 😄

**Review by krzdabrowski — APPROVED** [review]

**Pull request merged**